ReactViewControl.Avalonia/ReactViewRender.Avalonia.cs

```diff
@@ -46,6 +49,14 @@ partial class ReactViewRender : Control {
             }
         }
 
+        private static void OnHiddenWindowClosing(object sender, CancelEventArgs e) {
+            e.Cancel |= ((IClassicDesktopStyleApplicationLifetime)Application.Current.ApplicationLifetime).Windows.Count() != 0;
+            if (!e.Cancel) {
+                hiddenWindow.Closing -= OnHiddenWindowClosing;
```

Comment:
this whole if is not needed... the window should live for the entire app lifetime

Reply:
removed. Yeah, it never hit this one anyway...